mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -235,6 +235,14 @@ def create_experiment(self, name, artifact_location=None):
                     lifecycle_stage=LifecycleStage.ACTIVE,
                     artifact_location=artifact_location,
                 )
+                experiment_tags_dict = {}
+                if experiment_tags:
+                    for tag in experiment_tags:
+                        experiment_tags_dict[tag.key] = tag.value
+                experiment.tags = [
+                    SqlExperimentTag(key=key, value=value)
+                    for key, value in experiment_tags_dict.items()
+                ]
```

Comment:
Do we need to convert the list to a dictionary and convert the dictionary back to a list?

Reply:
Oh my god, I just realized that 😂 I copied this from the `create_run` code so I will optimize that one as well